pkgs/applications/misc/synergy/default.nix

```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = lib.optional (!withGUI) "-DSYNERGY_BUILD_LEGACY_GUI=OFF";
 
-  nativeBuildInputs = [ cmake ] ++ lib.optional withGUI wrapQtAppsHook;
+  nativeBuildInputs = [ cmake pkg-config ] ++ lib.optional withGUI wrapQtAppsHook;
```

Comment:
was I incorrect in adding pkg-config here and is that what is potentially breaking it? How do I test building the without gui version from source with my changes?

Reply:
No, that's not what's breaking it. Upstream changes made Qt a required dependency whether the GUI is built or not, so the solution is to always add `wrapQtAppsHook` to the list.

Reply:
Thank you!